.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
```

Comment:
- [ ] Kindly, go to the frontend [review] (https://github.com/MasumaJaffery/Car-Rent_App/pull/36#pullrequestreview-1748243004)to read the comments.